examples/np-rainbow-dynamic.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * This example shows how to use node-pixel to cycle colors through your
+ * NeoPixel strip using Johnny-Five.
+ * https://vimeo.com/131618207
+ */
```

Comment:
Add a note here around authorship - created by @pierceray in June 2015, good to have attribution